mlrun/runtimes/funcref.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2018 Iguazio
```

Comment:
Let's call the file `function_reference`

Reply:
ok

Reply:
fixed